### ISSUE TITLE
Can't set Breakpoint from the Breakpoint bar at the side

### ISSUE THREAD

**Issue by joelmoniz** [processing/processing#2835]
To reproduce this error, type some code in the Editor (for example:

```
void setup() {
  size(400,400);
}

void draw() {
  clear();
  delay(500);
  rect(random(20,40),random(20,40),random(20,40),random(20,40));
  ellipse(random(110,130),random(110,130),random(20,40),random(20,40));
}
```

Then, go to Debug>Show Debug Toolbar.
The Debug toolbar and the Breakpoint bar appear. A single click on the Breakpoint bar has no effect, while double clicking it sets and clears the breakpoint, causing the following log to appear (when testing with Eclipse, for example, in Eclipse's console window):

```
Aug 31, 2014 6:41:08 PM processing.mode.experimental.LineBreakpoint <init>
INFO: LBP Created test.pde:5 class: null
Aug 31, 2014 6:41:08 PM processing.mode.experimental.Debugger setBreakpoint
INFO: set breakpoint on line test.pde:5
Aug 31, 2014 6:41:08 PM processing.mode.experimental.Debugger removeBreakpoint
INFO: removed breakpoint test.pde:5
```

**Comment by shiffman** [comment]
I believe this can be closed as it is working now and #3093 covers the discussion.

**Comment by benfry** [comment]
Yeah, there was a weird bug that has mouse presses coming through twice. Not sure if it's an OS X thing or a larger bug that's being hidden.
